test/e2e/service_latency.go

```diff
@@ -119,7 +119,7 @@ var _ = Describe("Service endpoints latency", func() {
 func runServiceLatencies(f *Framework, inParallel, total int) (output []time.Duration, err error) {
 	cfg := RCConfig{
 		Client:       f.Client,
-		Image:        "gcr.io/google_containers/pause:1.0",
+		Image:        "beta.gcr.io/google_containers/pause:1.0",
```

Comment:
No, not yet.  Not sure what the update process is for these images?

Seeing the new linked issue, I'd be happy to consolidate these references under a new tag pushed it with v2?  That way we don't have to alter any existing tags, which is likely better.